test/backend/test_onnx_backend.py

```diff
@@ -32,6 +32,21 @@
 if 'TRAVIS' in os.environ:
   backend_test.exclude('test_vgg19')
 
+major, minor, revision = map(int, onnx.version.version.split("."))
+if major == 1 and minor < 2:
```

Comment:
nit: maybe ~1.1 and opset < 6 can be set to legacy version (add version check return true false)?
appears too many times... 
It's okay to keep it anyway.